plugins/streaming/impls/jms/src/main/java/org/pentaho/di/trans/step/jms/context/ActiveMQProvider.java

```diff
@@ -60,57 +68,82 @@
   protected String buildUrl( JmsDelegate delegate ) {
     StringBuilder finalUrl = new StringBuilder( delegate.amqUrl.trim() );
 
+    if ( log.isDebug() ) {
```

Comment:
I wouldn't wrap a block in log.isDebug(), unless it's potentiallly expensive.  It just adds noise otherwise.

Reply:
So the guidance here on log statements isn't as absolute as it reads?  I agree with you, just want to be sure I'm following the right style.

https://wiki.pentaho.com/display/EAI/Development+Guidelines